src/plugins/score-lib-process/Process/Script/ScriptEditor.cpp

```diff
@@ -51,6 +51,7 @@ ScriptDialog::ScriptDialog(
   connect(ce, &QCodeEditor::livecodeTrigger, this, &ScriptDialog::on_accepted);
   connect(bbox, &QDialogButtonBox::accepted, this, &ScriptDialog::on_accepted);
   connect(bbox, &QDialogButtonBox::rejected, this, &QDialog::reject);
+  connect(bbox->button(QDialogButtonBox::Close),&QPushButton::clicked,this,[=]{this->close();});
```

Comment:
Actually you can just do

     connect(bbox->button(QDialogButtonBox::Close),&QPushButton::clicked,
             this, &QDialog::close);

no ?

Reply:
Absolutely, made the changes.